public/js/components/App.js

```diff
@@ -5,27 +5,22 @@ const { bindActionCreators } = require("redux");
 const { cmdString } = require("../utils/text");
 const classnames = require("classnames");
 const actions = require("../actions");
-const { isFirefoxPanel } = require("../feature");
+const { isFirefoxPanel } = require("devtools-config");
 const { getSources, getSelectedSource } = require("../selectors");
 const { endTruncateStr } = require("../utils/utils");
 const { parse: parseURL } = require("url");
 
-const { KeyShortcuts } = require("devtools-sham/client/shared/key-shortcuts");
+const { KeyShortcuts } = require("devtools-sham-modules");
 const shortcuts = new KeyShortcuts({ window });
 
 require("./App.css");
-require("devtools/client/shared/components/splitter/SplitBox.css");
-
-// Using this static variable allows webpack to know at compile-time
-// to avoid this require and not include it at all in the output.
-if (process.env.TARGET !== "firefox-panel") {
-  require("../lib/themes/light-theme.css");
-}
```

Comment:
moved to the Root.js component, but that can be figured out later with the second phase where come up with the right API